docs/UserGuide.md

```diff
@@ -3,10 +3,46 @@ layout: page
 title: User Guide
 ---
 
-App-Ointment is a desktop app for for managing and scheduling patient appointments, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, App-Ointment can get your contact management tasks done faster than traditional GUI apps.
+App-Ointment is a desktop app for managing and scheduling patient appointments, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, App-Ointment can get your contact management tasks done faster than traditional GUI apps.
+
+## Table of Contents 
+1. Quick start
```

Comment:
I think you need to take a look at our [current website UG](https://ay2021s2-cs2103-w17-2.github.io/tp/UserGuide.html).

The table of contents is populated automatically with kramer, there is no need to replace the dynamic TOC with a static one.